.clang-format

```diff
@@ -0,0 +1,58 @@
+---
+Language: Cpp
+BasedOnStyle: LLVM
+
+# Indentation
+IndentWidth: 4
+TabWidth: 4
+ContinuationIndentWidth: 8
+AccessModifierOffset: -4
+#IndentPragmas: true # requires v12.0
+
+# Includes
+# group ordering is:
+#   1. main header
+#   2. double-quotes includes
+#   3. caesar includes
+#   4. all other angle bracket includes
+SortIncludes: true
+IncludeBlocks: Regroup
+IncludeIsMainSourceRegex: '(\.cu)$'
+IncludeCategories:
+    - Regex: '^"'
+      Priority: 1
+    - Regex: '^<caesar/'
+      Priority: 2
+
+# Line breaks
+AlwaysBreakTemplateDeclarations: Yes
+AlwaysBreakAfterReturnType: All
+ConstructorInitializerAllOnOneLineOrOnePerLine: true
+BinPackParameters: false
```

Comment:
This is probably the biggest divergence from isce3. I think I tend to prefer to use more vertical space than you guys do. Feel free to veto ;)

Reply:
that's very close to what i normally do, with one base class constructor per line, followed by one data member per line. the only difference is that i break them out this way even if they would fit. so this is just fine with me

Reply:
I prefer this actually, if the parameters are filling up too much vertical space, that's a cue to merge some of them into structs/tuples :)